src/test/java/org/cactoos/scalar/AvgOfTest.java

```diff
@@ -51,213 +51,313 @@ public void withEmptyCollection() {
     }
 
     @Test
-    public void withIntegerCollection() {
+    public void withIntCollectionIntValue() {
```

Comment:
@allanws Have you looked at other frameworks that work which values they used to test how calculate an average? Almost all tests are using only 1, 2, 3 and 4. It's important check with other numbers.